signing_service/signing_service/tests/test_signing_service_methods.py

```diff
@@ -275,6 +276,14 @@ def test_that_argument_parser_should_parse_parameters_if_passed_files_exist(self
         os.remove(ethereum_private_key_tmp_file)
         os.remove(signing_service_private_key_tmp_file)
 
+    def test_that_argument_parses_should_fail_gracefully_when_unable_to_decode_base64_value(self):
+        sys.argv += [
+            '--concent-cluster-host', '127.0.0.1',
+            '--concent-public-key', self.concent_public_key_encoded[:-1],
+        ]
+        with self.assertRaises(Base64DecodeError):
+            _parse_arguments()
+
```

Comment:
please mock ` logger.error`  in `utils` and ensure, that `Base64DecodeError` with **right message** is called. For now `Base64DecodeError` is used in only one place, but in future in may change and we should be 100% sure we are expecting exception from right place

Reply:
Done